share/spack/compiler/f77

```diff
@@ -0,0 +1 @@
+fc
```

Comment:
(question) is this properly formatted? How is it used? It looks to me like `default_try_file_for_lang` would return the wrong thing for `"f77"`.

Reply:
It was supposed to be a link to the file in `share/spack/compiler/fc` as I wasn't sure which Fotran standard to use for `fc` so I just used f77. I have 0 experience with Fortran, so could use some guidance as to whether this is the correct thing to do